### DISCUSSION TITLE
Drive.delete

### DISCUSSION THREAD

**Discussion by dmeinke-etc** [adonisjs/core#3504] [Help]
`static async DeleteFile(filePath){
    console.log(filePath)
    await Drive.use('s3').delete(filePath)
  }`

That's a method in my UploadFile class and filePath is valid, doesn't throw any errors but the file remains on Digital Ocean spaces

Seems pretty simple. Was the delete not meant to work with DO spaces?

**Comment by RoyalVICTOR-pro** [comment]
Hi @dmeinke-etc ! 
I have the same issue with Digital Ocean. I can upload file but I can't delete them. I don't have any errors but the file is still there. 
Did you find a solution ? 
Or anyone else ?

**Comment by vitortoigo** [comment]
Same problem

**Reply by RoyalVICTOR-pro** [reply]
@vitortoigo I managed to fix that on my end. It was due to the path of my files where I had left a ./ at the beginning. Removing it solved my problem.

**Reply by RoyalVICTOR-pro** [reply]
![image](https://github.com/adonisjs/core/assets/145127368/2dcaed28-fb7e-47f0-a96f-1ecf57a1679b)

**Reply by RoyalVICTOR-pro** [reply]
I hope this will help.

**Reply by vitortoigo** [reply]
Thx for the information. I'll try this.
